com.github.sdbg.debug.core/src/com/github/sdbg/debug/core/internal/util/DebuggerUtils.java

```diff
@@ -80,39 +80,7 @@ public static String demangleVmName(String name) {
       return null;
     }
 
-    int atIndex = name.indexOf('@');
```

Comment:
Ok cool, I didn't know that.

On Sun, Jan 18, 2015 at 7:47 PM, ivmarkov <notifications@github.com> wrote:

> In
> com.github.sdbg.debug.core/src/com/github/sdbg/debug/core/internal/util/DebuggerUtils.java
> <https://github.com/sdbg/sdbg/pull/106#discussion-diff-23135158>:
>
> > @@ -80,39 +80,7 @@ public static String demangleVmName(String name) {
> >        return null;
> >      }
> >
> > -    int atIndex = name.indexOf('@');
>
> @ is not an allowed symbol in javascript/java identifiers. However all of
> $ _ 0-9 are allowed symbols which (even though unlikely) may appear in an
> actual variable/field name.
>
> Anyway, if you are not willing to pursue the logical structure path, I'll
> give it a try anyway. I may merge your thing as a very short term solution
> tomorrow...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sdbg/sdbg/pull/106/files#r23135158>.
>